arch/AArch64/AArch64Mapping.c

```diff
@@ -341,10 +343,13 @@ static const char * const reg_name_maps[] = {
 const char *AArch64_reg_name(csh handle, unsigned int reg)
 {
 #ifndef CAPSTONE_DIET
-	if (reg >= ARR_SIZE(reg_name_maps))
-		return NULL;
+	if (reg < ARR_SIZE(reg_name_maps))
+		return reg_name_maps[reg];
+	SysReg *Reg = lookupSysRegByEncoding(reg);
```

Comment:
the variable should be declared at beginning of code block, not in the middle

Reply:
>the variable should be declared at beginning of code block, not in the middle

Oh, I haven't PR a C project before sorry for this mistake.